core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceTest.java

```diff
@@ -482,7 +533,7 @@ public void testAnnotatedHttpService() throws Exception {
             }
             // Run case 6 but with not-mapped HTTP method (Post).
             try (CloseableHttpResponse res = hc.execute(new HttpPost(rule.httpUri("/2/string/blah")))) {
-                assertThat(res.getStatusLine().toString(), is("HTTP/1.1 405 Method Not Allowed"));
+                assertThat(res.getStatusLine().toString(), is("HTTP/1.1 404 Not Found"));
```

Comment:
It was hard on the last PR, but it became easier on this PR, I think. I will make a fix.